finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/netty4/Netty4DispatcherBase.scala

```diff
@@ -187,7 +187,7 @@ trait Netty4DispatcherBase[SendMsg <: Message, RecvMsg <: Message] {
   }
 
   protected[this] val onTransportClose: Throwable => Unit = { e =>
-    log.debug(e, "[%s] transport closed", prefix)
+    log.debug("[%s] transport closed", prefix)
```

Comment:
Yeah, works for me. So something like:

```
    log.debug("[%s] transport closed: %s", prefix, e)
```

Which will log:

```
D 0714 22:21:54.271 UTC THREAD92: [S L:/127.0.0.1:4140 R:/127.0.0.1:60520] transport closed: com.twitter.finagle.ChannelClosedException: ChannelException at remote address: /127.0.0.1:60520. Remote Info: Not Available
```

I think that's sufficiently not scary enough when logged at the debug log level. How do folks feel about me making this change throughout the finagle/h2 project?